drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -162,6 +164,8 @@ def affiliated_projects
 
     # Build OpenStructs to resolve as UnitTypeCapacity
     def unit_types
+      return {} unless current_permission?(entity: object, permission: :can_view_units)
```

Comment:
If there are values, this method returns an array. I guess it's not causing problems, but I think it's clearer to return the same type `[]` instead of an empty object.